environments/staging/postgresql.yml

```diff
@@ -1,33 +1,24 @@
-postgresql_dbs:
-  - django_alias: default
-    name: "{{commcarehq_main_db_name}}"
+USE_PARTITIONED_DATABASE: True
```

Comment:
Thoughts on defaulting this to `True` and only specifying `False` when necessary? I think only swiss has this as false and if we could move that, we could get rid of some branching logic in our form processor code to make it a bit cleaner and also reduce our tests and remove the special partitioned travis job.

@snopoke This feels a bit tech-debty thoughts on having "moving swiss to partitioned db" on that backlog?

Reply:
Sure. Feel free to put it on there.

Reply:
Okay, good suggestion. I switched the default to True, and then also took the liberty of renaming the variable to `SEPARATE_FORM_PROCESSING_DBS` (but leaving it `USE_PARTITIONED_DATABASE` in `localsettings.py`).